extensions/variants.go

```diff
@@ -218,6 +222,20 @@ func parseFuncName(compoundName string) (name string, args ArgumentList) {
 	return name, args
 }
 
+func minArgumentCount(paramTypeList ArgumentList, variadicBehavior *VariadicBehavior) int {
+	if variadicBehavior == nil {
+		return len(paramTypeList)
+	}
+	return variadicBehavior.Min
+}
+
+func maxArgumentCount(paramTypeList ArgumentList, variadicBehavior *VariadicBehavior) int {
+	if variadicBehavior == nil {
+		return len(paramTypeList)
```

Comment:
we are calling `MaxArgumentCount` in the tests, but somehow this entire function isn't covered by the tests according to codecov, how is that?

Reply:
same as previous comment's response